README.md

```diff
@@ -105,6 +105,12 @@ module.exports = function (grunt) {
 };
 ```
 
+or via the CLI
+
+```sh
+$ bower-ls js css
```

Comment:
I though of that, too, but the cli really only lists the components, so `ls` seems like a good convention to keep with. Maybe an alias for `bower-files` would be good to put in. Thoughts?